modules/@apostrophecms/piece-type/index.js

```diff
@@ -1060,6 +1060,84 @@ module.exports = {
             });
           }
         }
+      },
+
+      localize: {
+        usage: 'Add draft version documents for each locale when a module has the "localized" option.' +
+        '\nExample: node app [moduleName]:localize',
+        async task() {
+          if (self.options.localized) {
+            console.log('Adding draft version to documents');
+            const locales = Object.keys(self.apos.i18n.locales);
+
+            await self.apos.migration.eachDoc({ type: self.name }, async doc => {
+              await self.apos.doc.db.removeOne({ _id: doc._id });
+
+              if (doc.aposDocId && !doc._id.endsWith('published') && !doc._id.endsWith('draft')) {
```

Comment:
Here you think about not tampering with docs that are somehow already localized if you come across them, which is a good idea, but on the previous line you still remove them. Make this check up front so you preserve them.